client/client.go

```diff
@@ -22,11 +23,11 @@ func IsNotification(r *http.Request) bool {
 
 // Query is a helper for initiating a request on the /1.0 endpoint. This function should be used for all client
```

Comment:
I think the comment here is now outdated

Reply:
Good catch, thanks :)